app/imports/schemas/newAdminUserSchema.js

```diff
@@ -9,6 +9,16 @@ export const NewAdminUserSchema = new SimpleSchema({
     allowedValues: [
       'Committee',
       'Admin'
-    ]
-  }
+    ],
+    uniforms: {
+      options: function () {
+        // This use " to keep literal values
+        return [
+          {value: "", label: ""},
+          {value: "Admin", label: i18n.__('admin.users.addUser.role.admin')},
```

Comment:
maybe the value should then be lowercase as the method that uses this form has to lowercase the string so it matches the roles set